Description: Avoid forked code in the build system
 This replaces omkdirhier and omkdepend which are source-level copies
 of other projects.  They are only used during the build system and
 can be easily replicated using standard OS and compiler programs.
Forwarded: http://www.omniorb-support.com/pipermail/omniorb-list/2002-October/022062.html
Author: Bastian Blank <waldi@debian.org>
Last-Update: 2017-06-28

--- a/mk/beforeauto.mk.in
+++ b/mk/beforeauto.mk.in
@@ -68,12 +68,10 @@
     TOOLBINDIR = $(TOP)/$(BINDIR)
 endif
 
-OMKDEPEND        = $(TOOLBINDIR)/omkdepend
 OMNIORB_IDL_ONLY = $(TOOLBINDIR)/omniidl -bcxx
 
 else
 
-OMKDEPEND        = @OMKDEPEND@
 OMNIORB_IDL_ONLY = @OMNIIDL@ -bcxx
 
 endif
@@ -708,7 +706,7 @@
 CC              = @CC@
 CXX             = @CXX@
 RANLIB		= @RANLIB@
-MKDIRHIER	= $(BASE_OMNI_TREE)/bin/scripts/omkdirhier
+MKDIRHIER	= mkdir -p
 INSTLIBFLAGS	= -m 0644
 INSTEXEFLAGS	= -m 0755
 CP		= cp
@@ -716,8 +714,8 @@
 CPP		= @CPP@
 RMDIRHIER	= $(RM) -rf
 
-CXXMAKEDEPEND   = $(OMKDEPEND) -D__cplusplus
-CMAKEDEPEND     = $(OMKDEPEND)
+CXXMAKEDEPEND   = @CXX@ -M -MD -MG
+CMAKEDEPEND     = @CC@ -M -MD -MG
 
 
 #
@@ -786,8 +784,6 @@
 #
 
 ifdef Compiler_GCC
-CMAKEDEPEND     += -D__GNUC__
-CXXMAKEDEPEND   += -D__GNUG__ -D__GNUC__
 CXXOPTIONS       = -Wall -Wno-unused -fexceptions @EXTRA_GCC_CXXFLAGS@
 EgcsMajorVersion = 1
 EgcsMinorVersion = 1
--- a/src/tool/dir.mk
+++ b/src/tool/dir.mk
@@ -7,7 +7,7 @@
 endif
 
 ifdef UnixPlatform
-SUBDIRS = omkdepend omniidl
+SUBDIRS = omniidl
 endif
 
 ifdef Win32Platform
